hw3/simple_router/lib/simple_router.rb

```diff
@@ -0,0 +1,22 @@
+require 'rack'
+require 'oj'
+require 'simple_router/version'
+require 'simple_router/router'
+require 'simple_router/controller'
+
+module SimpleRouter
+  class TestsController < Controller
```

Comment:
you can place test application to `spec/support/test_app.rb` or in `spec/fixtures/test_app.rb` and require it in spec helper